### DISCUSSION TITLE
Fast Method Instance.

### DISCUSSION THREAD

**Discussion by DarkAlchy** [TheLastBen/fast-stable-diffusion#245] [General]
You said on Reddit to "photo of X kword" with X being the Instance of man, woman, etc... but in all the other repos I have used I had to tell it the instance before encoded while this one I was never asked that.  What is X if I was not able to tell it, or is it really not needed?

**Comment by TheLastBen** [comment]
if you trained it on a man the prompt in A1111 would be : 
.... photo of Man kword

in the training you don't have to specify MAN

**Reply by DarkAlchy** [reply]
Thank you.  To test what was going on I said woman kword, and it gave the dude breasts, lol.

Thanks again.

**Reply by TheLastBen** [reply]
happy to help
